src/Plugin/Field/FieldFormatter/StrawberryMetadataTwigFormatter.php

```diff
@@ -152,25 +152,31 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
     }
 
     return [
-      'label' => [
-        '#type' => 'string',
-        '#title' => $this->t('Public facing Label for this Metadata Display'),
-        '#default_value' => $this->getSetting('label'),
+      'cutomtext' => [
+        '#type' => 'item',
+        '#markup' => '<h3>Use this form to select the template for your metadata.</h3><p>Several templates such as MODS 3.6 and a simple Object Description ship with Archipelago. To design your own template for any metadata standard you like, visit /metadatadisplay/list, and add it. </p>',
```

Comment:
Also, can you link the URL here? That would be great!

Reply:
Same here. Could you please on monday finish this pull? thanks

Reply:
@DiegoPino Got it, looking at this now!

Reply:
@marlo-longley wondering if you could like /metadatadisplay/list to an URL here? I left a comment about this on Nov 6. All the rest is perfect. If you can push that change i can merge today. Thanks!!